win32ss/user/user32/windows/cursoricon.c

```diff
@@ -1155,6 +1372,12 @@ BITMAP_LoadImageW(
         ResSize = SizeofResource(hinst, hrsrc);
     }
 
+ 
+    if (pbmi->bmiHeader.biCompression == BI_BITFIELDS &&
+        pbmi->bmiHeader.biBitCount == 32 &&
+        pbmi->bmiHeader.biSizeImage + pbmi->bmiHeader.biSize + 12 != ResSize)
```

Comment:
What is `12`?

Reply:
Please see PR #5942 and commit [116c0cd](https://github.com/reactos/reactos/commit/116c0cd).
This is the space for 3 DWORD's if this is a BI_BITFIELDS bmp in resource files.
This is different between MSVC (using rc.exe) versus GCC (using windres.exe) builds.
Now this shows up in this file at about Line number 520 and following.

Reply:
Fixed.